docs/en/engines/table-engines/mergetree-family/replacingmergetree.md

```diff
@@ -114,25 +114,31 @@ CREATE OR REPLACE TABLE myThirdReplacingMT
     `is_deleted` UInt8
 )
 ENGINE = ReplacingMergeTree(eventTime, is_deleted)
-ORDER BY key;
+ORDER BY key
+SETTINGS allow_experimental_replacing_merge_with_cleanup = 1;
 
 INSERT INTO myThirdReplacingMT Values (1, 'first', '2020-01-01 01:01:01', 0);
-INSERT INTO myThirdReplacingMT Values (1, 'first', '2020-01-01 01:01:01', 1); 
+INSERT INTO myThirdReplacingMT Values (1, 'first', '2020-01-01 01:01:01', 1);
 
 select * from myThirdReplacingMT final;
 
 0 rows in set. Elapsed: 0.003 sec.
 
--- delete rows with is_deleted
-OPTIMIZE TABLE myThirdReplacingMT FINAL CLEANUP; 
-
-INSERT INTO myThirdReplacingMT Values (1, 'first', '2020-01-01 00:00:00', 0);
```

Comment:
Why is it better? As I understand it shows that if insert value after something is deleted it remains.

Reply:
You're right. I thought the example would work the same with regular RMT, but I missed that `eventTime` was smaller in this last insert. I'll rollback to the previous example since it was more concise.

Reply:
No problem. Maybe it is better to split examples to make it better to understand why it is written that way :-)